index.html

```diff
@@ -324,15 +324,15 @@
     [[!HTML5]]</p>
     <pre class='idl'>
 partial interface Window {
-  unsigned long requestIdleCallback(IdleRequestCallback callback, optional IdleRequestOptions options);
-  void cancelIdleCallback(unsigned long handle);
+  [Exposed=Window] unsigned long requestIdleCallback(IdleRequestCallback callback, optional IdleRequestOptions options);
+  [Exposed=Window] void cancelIdleCallback(unsigned long handle);
```

Comment:
These two are unneeded as `Window` itself is already restricted.

Reply:
Done